msticpy/common/keyvault_client.py

```diff
@@ -24,17 +24,31 @@
     VaultProperties,
 )
 
+from IPython.display import display, HTML
 from msrestazure.azure_exceptions import CloudError
 
 from .._version import VERSION
 from .azure_auth_core import az_connect_core
-from .exceptions import MsticpyKeyVaultConfigError, MsticpyKeyVaultMissingSecretError
+from .exceptions import (
+    MsticpyKeyVaultConfigError,
+    MsticpyKeyVaultMissingSecretError,
+    MsticpyUserConfigError,
+)
 from .keyvault_settings import KeyVaultSettings
-from .utility import export
+from .utility import export, is_ipython
 
 __version__ = VERSION
 __author__ = "Matt Richard, Ian Hellen"
 
+_KV_CLIENT_AUTH_ERROR = [
+    "Please use Azure CLI authentication:",
+    "1. Add 'cli' to the list of authentication methods in Azure/auth_methods"
+    " in msticpyconfig.yaml ",
+    "2. run 'az login` at the start of your session.",
+    "3. Re-run the MSTICPy function that failed with this error.",
+    "",
```

Comment:
Do we want to include a link to https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli in this message?

Reply:
Good idea (but without the "en-us" ;-) )